### PR TITLE
PLF-7091 Feature/jbosseap 7

### PR DESCRIPTION
**Pull request by boubaker**

### CODE REVIEW

File: component/web/controller/src/main/java/org/exoplatform/web/PortalHttpServletResponseWrapper.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+  public boolean isWrapMethods() {
+    return wrapMethods;
+  }
+
+  public void setWrapMethods(boolean wrapMethods) {
+    this.wrapMethods = wrapMethods;
+  }
+
+  public void commit() throws IOException {
```
Comment (thomasdelhomenie) [inline component/web/controller/src/main/java/org/exoplatform/web/PortalHttpServletResponseWrapper.java]: super.commit() is never called. Is it intended ?

Comment (boubaker) [inline component/web/controller/src/main/java/org/exoplatform/web/PortalHttpServletResponseWrapper.java]: In fact, super.commit() doesn't exist, this is a custom method added to commit changes on original response object at the end of HTT Request lifecycle.

File: component/web/pom.xml [hunk 2]
```diff
@@ -37,7 +37,6 @@
     <module>server</module>
     <module>resources</module>
     <module>api</module>
-    <module>security-jboss</module>
```
Comment (boubaker) [inline component/web/pom.xml]: In this module a valve was added to make the SSO between webapps works. In JBoss EAP 7, it's made by configuration.

File: component/web/controller/src/main/java/org/exoplatform/web/PortalHttpServletResponseWrapper.java [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpServletResponseWrapper;
+
+public class PortalHttpServletResponseWrapper extends HttpServletResponseWrapper {
```
Comment (boubaker) [inline component/web/controller/src/main/java/org/exoplatform/web/PortalHttpServletResponseWrapper.java]: done

**Pull request merged**